src/services/notehub/NotehubAttributeStore.ts

```diff
@@ -0,0 +1,27 @@
+import { AttributeStore } from "../AttributeStore";
+import { NotehubAccessor } from "./NotehubAccessor";
+
+export class NotehubAttributeStore implements AttributeStore {
+  constructor(private accessor: NotehubAccessor) {}
+
+  async updateSensorName(gatewayUID: string, macAddress: string, name: string) {
+    const { body } = await this.accessor.getConfig(gatewayUID, macAddress);
```

Comment:
Why do you have to get the config before setting it?

Reply:
I believe it's because the API only allows you to set the whole contents of a note in a `.db` file.  Incremental update would have been useful here. 

https://dev.blues.io/reference/notecard-api/note-requests/#note-update

Reply:
Ah gotcha. In that case a comment in the code here might be a good idea.

Reply:
Or we could put them in two different files or notes. I think it's a bit much to ask Notehub to also allow incremental updates of note bodies. Like, how many layers down do they need to go to satisfy us?